alembic/versions/700e63da6309_airdroptables.py

```diff
@@ -0,0 +1,119 @@
+    sa.Column('address', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('org_name', sa.VARCHAR(length=256), nullable=False),
+    sa.Column('token_name', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('token_type', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('contract_address', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('portal_link', sa.VARCHAR(length=256), nullable=True),
+    sa.Column('documentation_link', sa.VARCHAR(length=256), nullable=True),
+    sa.Column('description', sa.TEXT(), nullable=True),
+    sa.Column('github_link_for_contract', sa.VARCHAR(length=256), nullable=True),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.PrimaryKeyConstraint('row_id'),
+    sa.UniqueConstraint('row_id')
+    )
+    op.create_table('airdrop_window',
+    sa.Column('row_id', sa.INTEGER(), autoincrement=True, nullable=False),
+    sa.Column('airdrop_id', sa.INTEGER(), nullable=False),
+    sa.Column('description', sa.TEXT(), nullable=True),
+    sa.Column('registration_required', mysql.BIT(), nullable=True),
+    sa.Column('registration_start_period', mysql.TIMESTAMP(), nullable=False),
+    sa.Column('registration_end_period', mysql.TIMESTAMP(), nullable=False),
+    sa.Column('snapshot_required', mysql.BIT(), nullable=True),
+    sa.Column('first_snapshot_at', mysql.TIMESTAMP(), nullable=True),
+    sa.Column('claim_start_period', mysql.TIMESTAMP(), nullable=False),
+    sa.Column('claim_end_period', mysql.TIMESTAMP(), nullable=False),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.PrimaryKeyConstraint('row_id')
+    )
+    op.create_table('airdropwindow_rules',
+    sa.Column('row_id', sa.BIGINT(), autoincrement=True, nullable=False),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('airdrop_window_id', sa.INTEGER(), nullable=False),
+    sa.Column('rule', sa.TEXT(), nullable=False),
+    sa.PrimaryKeyConstraint('row_id')
+    )
+    op.create_table('claim_history',
+    sa.Column('row_id', sa.BIGINT(), autoincrement=True, nullable=False),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('airdrop_window_id', sa.INTEGER(), nullable=False),
```

Comment:
Dont we need airdrop_id as well in this table?

Reply:
I was thinking another way by leveraging with SQL queries we get the `airdrop _id` but that will be a heavy operation  @raamb  


Added the `airdrop_id` in the latest commit.